test/unit/schema/postgres.js

```diff
@@ -5,10 +5,18 @@
 var tableSql;
 
 var PG_Client = require('../../../lib/dialects/postgres');
-var client    = new PG_Client({})
+var client    = new PG_Client({});
 
 var equal  = require('assert').equal;
 
+describe("PostgreSQL Config", function(){
+  it('check version', function(){
+    var version = 'v7.2.1-0';
```

Comment:
@RockyNiu I think @wubzz added that test while ago... maybe it random fails... it shouldn't be related to this PR